src/reference/antora/modules/ROOT/pages/cloudevents.adoc

```diff
@@ -0,0 +1,268 @@
+[[cloudevents]]
+= CloudEvents Support
+
+Spring Integration provides transformers (starting with version 7.0) for transforming messages into CloudEvent messages.
+It is fully based on the https://github.com/cloudevents/sdk-java[CloudEvents SDK] project.
+
+This dependency is required for the project:
+
+[tabs]
+======
+Maven::
++
+[source, xml, subs="normal", role="primary"]
+----
+<dependency>
+    <groupId>org.springframework.integration</groupId>
+    <artifactId>spring-integration-cloudevents</artifactId>
+    <version>{project-version}</version>
+</dependency>
+----
+
+Gradle::
++
+[source, groovy, subs="normal", role="secondary"]
+----
+compile "org.springframework.integration:spring-integration-cloudevents:{project-version}"
+----
+======
+
+[[cloudevent-transformers]]
+
+== CloudEvent Transformers
+
+[[tocloudeventstransformer]]
+=== ToCloudEventsTransformer
+
+The `ToCloudEventsTransformer` converts Spring Integration messages into CloudEvents compliant messages.
+This transformer provides  support for the CloudEvents specification v1.0 with configurable output format and defining attributes using `Expression` s and identifying extensions in the message headers via patterns.
+
+[[cloudevent-transformer-overview]]
+==== Overview
+
+The CloudEvents transformer (`ToCloudEventsTransformer`) converts messages to CloudEvents format.
```

Comment:
This is a bit misleading when we introduce a `FromClouldEventTransformer`.
Why do we need a separate `Overview` title?